### ISSUE TITLE
Can't retrieve summary on Shelley addresses

### ISSUE THREAD

**Issue by alexqrid** [input-output-hk/cardano-rest#75]
Endpoint `/api/addresses/summary` response with error when trying to query with shelley address like `addr1....`
```sh
$ curl localhost:8100/api/addresses/summary/addr1q8cs8j8rv43v59ggt9zhjhngltt7qnz47zc6ypysptzd5p8kf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6eslaw3az
{
  "Left": "Unable to decode address addr1q8cs8j8rv43v59ggt9zhjhngltt7qnz47zc6ypysptzd5p8kf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6eslaw3az."
}
```

**Comment by rhyslbw** [comment]
@krisajenkins 
```
echo addr1q8cs8j8rv43v59ggt9zhjhngltt7qnz47zc6ypysptzd5p8kf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6eslaw3az | cardano-address address inspect
{
    "stake_reference": "by value",
    "stake_key_hash": "f64bfb23ca8db2cef5d15152d2a4494771237118ddde9a3c9754b6b3",
    "address_style": "Shelley",
    "spending_key_hash": "f103c8e36562ca15085945795e68fad7e04c55f0b1a204900ac4da04",
    "network_tag": 1
}
```
https://github.com/input-output-hk/cardano-addresses

**Comment by rhyslbw** [comment]
Suspect this is related to https://github.com/input-output-hk/cardano-db-sync/issues/224

**Comment by ArturWieczorek** [comment]
This is fixed now (commit b182ef92a15606372df0e335a1cd647849f5a90c): 

`curl localhost:8100/api/addresses/summary/addr1q8cs8j8rv43v59ggt9zhjhngltt7qnz47zc6ypysptzd5p8kf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6eslaw3az`


```json
{
   "Right":{
      "caAddress":"addr1q8cs8j8rv43v59ggt9zhjhngltt7qnz47zc6ypysptzd5p8kf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6eslaw3az",
      "caType":"CPubKeyAddress",
      "caChainTip":{
         "ctBlockNo":4495234,
         "ctSlotNo":4587700,
         "ctBlockHash":"e1661f57301b17004e74590f9d0548710ded3ff07a97889006b1ff7d627085be"
      },
      "caTxNum":2,
      "caBalance":{
         "getCoin":"0"
      },
      "caTotalInput":{
         "getCoin":"200000000"
      },
      "caTotalOutput":{
         "getCoin":"200000000"
      },
      "caTotalFee":{
         "getCoin":"2368050"
      },
      "caTxList":[
         {
            "ctbId":"a9690b3de323a9aafe6cecf374447f7b5c219553ed2410a2c0a093d534fc94e5",
            "ctbTimeIssued":1596060071,
            "ctbInputs":[
               {
                  "ctaAddress":"DdzFFzCqrhtBT7Pz1ugHNxMLepDamZiySemAKSpNnNrjearFbsspsSAyfVmtnr5pFBTVoH3XEYkQ69aBqUzkiSd434gXiYeXsZsUYpDt",
                  "ctaAmount":{
                     "getCoin":"68500000"
                  },
                  "ctaTxHash":"b009672b9e6177e2a9cc191f0a9a7b8998804d36d81535cfc7500a5e09b34461",
                  "ctaTxIndex":1
               },
               {
                  "ctaAddress":"DdzFFzCqrhtBT7Pz1ugHNxMLepDamZiySemAKSpNnNrjearFbsspsSAyfVmtnr5pFBTVoH3XEYkQ69aBqUzkiSd434gXiYeXsZsUYpDt",
                  "ctaAmount":{
                     "getCoin":"200000000"
                  },
                  "ctaTxHash":"b240ef27c85c6bbac20b5c2c881908eb2be91d03f43730ef842342d93cb04727",
                  "ctaTxIndex":1
               },
               {
                  "ctaAddress":"DdzFFzCqrhtBT7Pz1ugHNxMLepDamZiySemAKSpNnNrjearFbsspsSAyfVmtnr5pFBTVoH3XEYkQ69aBqUzkiSd434gXiYeXsZsUYpDt",
                  "ctaAmount":{
                     "getCoin":"11899700000"
                  },
                  "ctaTxHash":"026e947920ad17eb2ef13efc8a4c1dd2fc48ab668c10beaa774a182c39549afb",
                  "ctaTxIndex":1
               }
            ],
            "ctbOutputs":[
               {
                  "ctaAddress":"addr1q8cs8j8rv43v59ggt9zhjhngltt7qnz47zc6ypysptzd5p8kf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6eslaw3az",
                  "ctaAmount":{
                     "getCoin":"200000000"
                  },
                  "ctaTxHash":"a9690b3de323a9aafe6cecf374447f7b5c219553ed2410a2c0a093d534fc94e5",
                  "ctaTxIndex":2
               },
               {
                  "ctaAddress":"addr1q8fmuw4tpxmktmls7lnjn2h67vmr398h003mgx4feqeamdhkf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6esf4nx87",
                  "ctaAmount":{
                     "getCoin":"68500000"
                  },
                  "ctaTxHash":"a9690b3de323a9aafe6cecf374447f7b5c219553ed2410a2c0a093d534fc94e5",
                  "ctaTxIndex":1
               },
               {
                  "ctaAddress":"addr1qyp7n677ha44xffxex5fcx8rfpkr7d0kp4g27kpu2xmclwhkf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6es7tty56",
                  "ctaAmount":{
                     "getCoin":"11899506207"
                  },
                  "ctaTxHash":"a9690b3de323a9aafe6cecf374447f7b5c219553ed2410a2c0a093d534fc94e5",
                  "ctaTxIndex":0
               }
            ],
            "ctbInputSum":{
               "getCoin":"12168200000"
            },
            "ctbOutputSum":{
               "getCoin":"12168006207"
            },
            "ctbFees":{
               "getCoin":"193793"
            }
         },
         {
            "ctbId":"a6c41b0a1ca4aa471515f6d19369830fd45316da1e6f77167034f99f2ab506ed",
            "ctbTimeIssued":1596103171,
            "ctbInputs":[
               {
                  "ctaAddress":"addr1q8cs8j8rv43v59ggt9zhjhngltt7qnz47zc6ypysptzd5p8kf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6eslaw3az",
                  "ctaAmount":{
                     "getCoin":"200000000"
                  },
                  "ctaTxHash":"a9690b3de323a9aafe6cecf374447f7b5c219553ed2410a2c0a093d534fc94e5",
                  "ctaTxIndex":2
               }
            ],
            "ctbOutputs":[
               {
                  "ctaAddress":"addr1q88fhf8v0tcckkw7xee66l3zeq5sg2uwmevazdkuqtvj8xlkf0aj8j5dkt80t5232tf2gj28wy3hzxxam6dre965k6esxhunym",
                  "ctaAmount":{
                     "getCoin":"197825743"
                  },
                  "ctaTxHash":"a6c41b0a1ca4aa471515f6d19369830fd45316da1e6f77167034f99f2ab506ed",
                  "ctaTxIndex":0
               }
            ],
            "ctbInputSum":{
               "getCoin":"200000000"
            },
            "ctbOutputSum":{
               "getCoin":"197825743"
            },
            "ctbFees":{
               "getCoin":"2174257"
            }
         }
      ]
   }
}
```
